pkg/users/local.js

```diff
@@ -57,13 +71,22 @@ function AccountsPage() {
             return Object.assign({}, account, details[i]);
         });
 
+    const groupsExtraInfo = sortGroups(
+        (groups || []).map(group => {
+            const userlistPrimary = accountsInfo.filter(account => account.gid === group.gid).map(account => account.name);
+            const userlist = group.userlist.filter(el => el !== "");
+            return ({ ...group, userlistPrimary, userlist, members: userlist.length + userlistPrimary.length, isAdmin: admins.includes(group.name) });
+        })
+    );
+
     if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groups || []} />;
-    } else
+        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221213-083353-49bdb546-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)